src/olympia/blocklist/tests/test_forms.py

```diff
@@ -140,3 +164,38 @@ def test_guids_must_exist_for_block_deletion(self):
         form.is_valid()
         with self.assertRaises(ValidationError):
             form.clean()
+
+
+class TestMultiAddForm(LegacySyncCheckMixin, TestCase):
+    def test_guid_must_exist_in_database(self):
+        data = {
+            'guids': 'any@thing',
+        }
+
+        form = MultiAddForm(data=data)
+        form.is_valid()
+        with self.assertRaises(ValidationError):
+            # any@thing doesn't exist
+            form.clean()
+
+        # but that check is bypassed for multiple guids
+        # (which are highlighted on the following page instead)
+        data = {
+            'guids': 'any@thing\nsecond@thing',
+        }
+
+        form = MultiAddForm(data=data)
+        addon_factory(guid='any@thing')
+        Block.objects.create(guid='any@thing', updated_by=user_factory())
+        addon_factory(guid='second@thing')
+        form.is_valid()
+        form.clean()  # would raise
```

Comment:
shouldn't it be `wouldn't raise`?

Reply:
as in, would raise if there was an error.

Reply:
ah ok, then maybe update the comment with the longer version :D